include/seqan3/core/algorithm/detail/algorithm_executor_blocking.hpp

```diff
@@ -156,17 +156,14 @@ class algorithm_executor_blocking
      * Also note that the third argument is used for deducing the algorithm result type and is otherwise
      * not used in the context of the class' construction.
      */
-    template <typename exec_policy_t = sequenced_policy>
-    //!\cond
-        requires is_execution_policy_v<exec_policy_t>
-    //!\endcond
     algorithm_executor_blocking(resource_t resource,
                                 algorithm_t algorithm,
                                 algorithm_result_t const SEQAN3_DOXYGEN_ONLY(result) = algorithm_result_t{},
-                                exec_policy_t const & SEQAN3_DOXYGEN_ONLY(exec) = seq) :
-        resource{std::views::all(resource)},
-        resource_it{std::ranges::begin(this->resource)},
-        algorithm{std::move(algorithm)}
+                                execution_handler_t && exec_handler = execution_handler_t{})
+        : exec_handler{std::move(exec_handler)},
```

Comment:
Before the colon was on the previous line, now it is on the new line. Is the code style change intended?